### ISSUE TITLE
Output as gif

### ISSUE THREAD

**Issue by mribeiro** [bsharper/hyper-capture#1]
Could this be used to create gif files?

**Comment by bsharper** [comment]
Yes it could. Currently the extension outputs video files which could be converted to a gif file. In fact, the 2nd gif on this page https://github.com/bsharper/hyper-capture was converted from a video to a gif.

You still have to do the video to gif conversion yourself, but there should be good information on how to do that online. Let me know if you have trouble finding it.

**Comment by mribeiro** [comment]
A good idea could be doing it automatically in hyper-capture 😜 JK.
Anytime I try to convert the webm to gif it gets a really poor color quality. Is this a limitation of webm to gif conversion? Sorry for coming up with this, maybe I should google a bit more 😄

**Comment by bsharper** [comment]
There is a way you can get around bad color quality. If you are using Linux or macOS, you can try the instructions below. If you are using Windows, you might be able to make it work with Windows if you're able to change the shell script into a .bat script (mostly changing the variables from $this to %this% and setting `palette="/tmp/palette.png"` to something like `palette="palette.png"`).

I'm pretty sure this is what I used: http://blog.pkh.me/p/21-high-quality-gif-with-ffmpeg.html

First, make sure you have `ffmpeg` installed somewhere. Then copy the code under the Usage section in the link above to a file called `gifenc.sh`. Then do `chmod +x gifenc.sh` and convert your video: `./gifenc.sh some_capture.web output.gif`.

If you don't want the GIF to be resized, changed the `filters` line to `filters="fps=15"`.

Let me know if this does the trick for you. I might add GIF conversion in the near future, I just need to make sure it works cross-platform.

**Comment by johhansantana** [comment]
is this still working? i tried adding it to hyper v2 using `hyper i hyper-capture` but `zsh: command not found: tooglecapture`
